.github/workflows/main.yml

```diff
@@ -23,6 +23,10 @@ jobs:
           POSTGRES_PASSWORD: postgres
           POSTGRES_DB: test
         options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5
+      cockroachdb:
+        image: timveil/cockroachdb-single-node:latest
```

Comment:
Let's try and use the official image here please - everything else looks good at a quick glance, but we'd want to rely on the official image in the builds.